source/python/idetect/interpreter.py

```diff
@@ -71,6 +71,29 @@ def get_absolute_date(relative_date_string, publication_date=None):
         return None
 
 
+def cleanup(text):
```

Comment:
Can you import this from model? We're already importing several things from there.

Reply:
Hah...good idea.
I initially set it up to import into model from interpreter, but ended up getting a circular import error and it didn't occur to me to do the other way round.  :thumbsup: